app/src/main/java/com/example/evently/ui/common/EventDetailsFragment.java

```diff
@@ -102,6 +107,28 @@ private void loadEventInformation(Event event) {
         binding.eventName.setText(event.name());
         binding.eventDescription.setText(event.description());
         binding.eventCategory.setText(event.category().toString());
+
+        // Sets the seat text:
+        String seatLimit = String.valueOf(event.selectionLimit());
+        binding.seatsText.setText(seatLimit);
+
+        // If we have a limit, it is shown
+        if (event.optionalEntrantLimit().isPresent()) {
+            binding.waitlistSeparator.setVisibility(View.VISIBLE);
+            binding.entrantLimit.setVisibility(View.VISIBLE);
+            // Sets the entrant limit text.
+            String entrantLimit = event.optionalEntrantLimit().get().toString();
+            binding.entrantLimit.setText(entrantLimit);
+        } else {
+            binding.waitlistSeparator.setVisibility(View.INVISIBLE);
+            binding.entrantLimit.setVisibility(View.INVISIBLE);
+        }
+
+        // Formats the date, and stores it in the selection date text
+        String formattedDate =
+                EVENT_DATE_TIME_FORMATTER.format(event.selectionTime().toInstant());
+        binding.selectionDateText.setText(formattedDate);
```

Comment:
This is great! Maybe you can add in the rest of the information as well? #166 

I'm thinking we'll replace the "location" thing with "organizer" (maybe a little icon for that too). We'll just show the organizer name there maybe?